app/Http/Controllers/Api/SessionController.php

```diff
@@ -115,11 +134,21 @@ public function getAllSession()
         return response()->json($session, 200);
     }
 
-    static public function getCurrentSessionId()
+    static public function getCurrentSession()
     {
-        $currentSession = Session::where('is_current', '1')->first();
+        $currentSession = Session::where(['is_current'=>'1','status'=>'Active'])->first();
+        if(!empty($currentSession)){
+            return $currentSession;
+        }
+        return response()->json(['error' => 'Session not found'], 404);
+
+    }
+    static public function getCurrentSessionId($value='')
```

Comment:
DRY! why not call the getCurrentSession() and point to the id prop to get the id. OR
though if you feel a new method is necessary then we can call getCurrentSession() inside getCurrentSessionId() and do the same.

Reply:
😺